monitoring/voltage-monitoring/src/main/java/com/farao_community/farao/monitoring/voltage_monitoring/VoltageMonitoringResult.java

```diff
@@ -21,9 +24,12 @@ public class VoltageMonitoringResult {
 
     public enum Status {
         SECURE,
+        UNSECURE,
```

Comment:
The status "UNSECURE" should not exist in this enum as the 3 possibilities are listed right below.

Reply:
Done !